### PR TITLE
Add Lara and Kain to calculators

### LINKED ISSUE #13: Lara needs to be added to hyper stat calc
See title. Lara missing from hyper stat optimizer.

### PR DESCRIPTION
**Pull request by NotTsunami**
Closes #13

### CODE REVIEW

**Comment by NotTsunami** [comment]
(Please read the description for #26 before merging)

File: LegionCalculator/src/hyper.js [hunk 1]
```diff
@@ -2420,7 +2445,8 @@ function getClassData(maple_class) {
             'attPercent': 4,
             'iedPercent': [44, 20],
             'dmgPercent': 20,
-            'bossPercent': 0
+            'bossPercent': 0,
+            'critDmg': 0
```
Comment (NotTsunami) [inline LegionCalculator/src/hyper.js:2449]: Forgot to apply this to the hyper stat optimizer as well. I can also extract this fix to a separate pull request if you'd like.

**Comment by brendonmay** [comment]
@NotTsunami Lara and Kain need to be added to the dropdown menus on the front-end for the user.

**Comment by NotTsunami** [comment]
> @NotTsunami Lara and Kain need to be added to the dropdown menus on the front-end for the user.

Where is the data for that located? I just did a simple search to match the class names and didn't see any other entries, so I assumed it was populated via the `getClass` functions (forgot exact name).

**Comment by brendonmay** [comment]
@NotTsunami I believe its in the HTML file.

**Pull request merged**